src/database/datatools/designday.py

```
@@ -0,0 +1,234 @@
+                                  ashrae_clear_sky_optical_depth_for_diffuse_irradiance_taud: Optional[float] = None,
+                                  sky_clearness: Optional[float] = None,
+                                  maximum_number_warmup_days: Optional[int] = None,
+                                  begin_environment_reset_mode: Optional[str] = None) -> None:
+    
+    
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "sizingperiod_designday"
+
+    sql = f"INSERT INTO {table_name} (name, latitude, longitude, architecture_type, month, day_of_month, day_type, max_drybulb_temperature, daily_drybulb_temperature_range, drybulb_temperature_range_modifier_type, drybulb_temperature_range_modifier_day_schedule_name, humidity_condition_type, wetbulb_or_dewpoint_at_maximum_drybulb, humidity_condition_day_schedule_name, humidity_ratio_at_maximum_drybulb, enthalpy_at_maximum_drybulb, daily_wetbulb_temperature_range, barometric_pressure, wind_speed, wind_direction, rain_indicator, snow_indicator, daylight_saving_time_indicator, solar_model_indicator, beam_solar_day_schedule_name, diffuse_solar_day_schedule_name, ashrae_clear_sky_optical_depth_for_beam_irradiance_taub, ashrae_clear_sky_optical_depth_for_diffuse_irradiance_taud, sky_clearness, maximum_number_warmup_days, begin_environment_reset_mode, datetime) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)